sonar-db/src/main/resources/org/sonar/db/user/UserMapper.xml

```diff
@@ -98,6 +98,14 @@
     </where>
   </select>
 
+  <select id="countByEmail" parameterType="String" resultType="long">
+    SELECT count(u.id)
+    FROM users u
+    <where>
```

Comment:
In fact it's not necessary an issue : If the user enter an existing email with some upper cases, then it will be considered as a different email.
Fabrice is thinking about that : if he'll think email should be strictly case insensitive then we'll create another ticket to work on that (update existing emails to set them in lower case, insert and search in lower case).